examples/Programmable Buttons/Programmed External Keys.ino

```diff
@@ -0,0 +1,68 @@
+/*
+  Programmed External Keys (Keyboard)
+  
+  For the Arduino Leonardo/any other arduino board with usb interface
+  
+  Reads a key from push button & acts as a CTRL+C (Copy).
+  Reads another key from push button & acts as a CTRL+V (Paste).
+  
+
+  The circuit:
+  
+  The push button has 2 connectors. 
+  We must connect gnd and the second will be connected to any I/O pin. 
+
+connect register between (button & gnd)& then set pinMode() as INPUT.
+
+  created 18 March 2020
+  
+  by Prathamesh Sahasrabhojane (TheShubham99)
+
+*/
+
+#include "Keyboard.h"
+
+const int btnPin[] = {2, 3, 4, 5};
+int pincount = 4;
+int btnState[] = {0, 0, 0, 0};
+int prevbtnState[] = {HIGH, HIGH, HIGH, HIGH};
+
+long lastDebounceTime[] = {0, 0, 0, 0};
+long debounceDelay = 50;
+void setup() {
+  for (int thisPin = pincount - 1; thisPin >= 0; thisPin--) {
+    pinMode(btnPin[thisPin], INPUT);
+    digitalWrite(btnPin[thisPin], HIGH);
+  }
+  Keyboard.begin();
+}
+
+// Place the actions needed after pin press here
+int outputAction(int currentButton) {
+    if (currentButton == 1) {
+      Keyboard.press(ctrlKey);
+      Keyboard.press('c');
+      delay(100);
+      Keyboard.releaseAll();
+    }
+    if (currentButton + 1 == 2) {
+      Keyboard.press(ctrlKey);
+      Keyboard.press('v');
+      delay(100);
+      Keyboard.releaseAll();
+    }
+}
```

Comment:
What happened to the other two buttons?

Reply:
yes, they are programmable but I haven't added the functionality for them. should I mention it in the comments or should I remove those keys?

Reply:
If you're not going to use them, remove them, since they will just cause confusion for beginners. If someone wants to expand on the sketch, they should be able to do that easily enough.

I see you started on that by changing `pincount` to 2, but you didn't remove the unused pins from `btnPin[]`, `prevBtnState[]`, `lastDebounceTime[]`. You didn't change the index numbers in `outputAction()`, which means the sketch doesn't even work now (because `currentButton` can never be 2). I guess you forgot the thing I said about thoroughly testing your work.

Reply:
Done.